FLOE-128

jobara opened the pull request:
Styling work for the simple editor integrating with the metadata panels.

http://issues.fluidproject.org/browse/FLOE-128

cindyli committed:
FLOE-115: Start on captions metadata panel.

jobara committed:
FLOE-115: In progress commit

cindyli committed:
FLOE-115: More progress with the captions metadata panel. The only issue is the refreshView() called at captions modelChanged in order to refresh the icon state creates keyboard inaccessible issue since the re-rendering causes the lose of focus. Trying sub-component to render the icon instead of using the protoTree decorator.

Commit:
Merge branch 'FLOE-112' into FLOE-128

cindyli committed:
FLOE-115: Completed the captions panel and its unit tests.

cindyli committed:
Merge branch 'master' into FLOE-115

cindyli committed:
Merge branch 'FLOE-115' into FLOE-125

cindyli committed:
FLOE-125: Integrated metadata panels with the simple editor.

Commit:
Merge branch 'master' into FLOE-128

Commit:
Merge remote-tracking branch 'cindyli/FLOE-125' into FLOE-128

cindyli committed:
FLOE-125: Removed metadata panels at restarting demo.

Commit:
FLOE-128:
- Added OpenSans-Light font
- Added font icon for metadata panel
- name spaced the demo
- styled the metadata panels
- edited the text for the metadata panel form options
- fixed the mobile layout and visual appearance

Commit:
FLOE-128: Changed the title of the demo. Changed "View HTML" to "View Output HTML". Changed the colour of the OK button on the toolbar to have more contrast.

jobara committed:
Merge branch 'master' into FLOE-128

Conflicts:
	demos/html/metadata.html

jobara committed:
Merge remote-tracking branch 'cindy/FLOE-125' into FLOE-128

jobara commented:
Depends on https://github.com/fluid-project/metadata/pull/8

michelled commented on demos/css/simpleEditor.css:
```diff
@@ -37,29 +45,35 @@ header {
-.fl-simpleEditor-toolbar {
+.fl-demo .fl-simpleEditor-toolbar {
     padding-top: 0.3em;
+    border-top-left-radius: 0.5em; /* At this resolution, just round these corners. */
```
is border radius something that we want to scale? this might be one case where we prefer pixels.

Commit:
FLOE-128: Removed the inline form styling for URL input field in toolbar, and replaced with Bootstrap grid layout. Also adjusted the text for the URL input label.

jobara committed:
Merge branch 'master' into FLOE-128

Commit:
Merge branch 'master' into FLOE-128

Commit:
FLOE-128: Modified layout to use more Bootstrap grids. Hopefully this will fix issues on mobile devices.

jobara committed:
Merge remote-tracking branch 'jhung/FLOE-128' into FLOE-128

jhung committed:
FLOE-128: Moved Video URL input and button into the same container so that button doesn't get cropped in Webkit browsers.

Added fluid width styling to Video URL text field so it scales better on smaller devices.

jobara committed:
Merge remote-tracking branch 'jhung/FLOE-128' into FLOE-128

jhung committed:
FLOE-128: Reduced the fluid URL text entry field width a little so the button doesn't wrap on mobile layouts.

jobara committed:
Merge remote-tracking branch 'jhung/FLOE-128' into FLOE-128

jhung commented:
This has diverged from the original FLOE-128 branch. I am closing this and resubmitting it with the proper branch.
